cics-bundle-maven-plugin/src/main/java/com/ibm/cics/cbmp/PackageCICSBundleMojo.java

```diff
@@ -75,13 +82,15 @@ static File createCICSBundle(
 			BundlePublisher bundlePublisher,
 			String classifier,
 			MavenProject project,
+			String finalName,
 			MavenProjectHelper projectHelper) throws MojoExecutionException {
 		try {
 			bundlePublisher.publishDynamicResources();
 			
 			ZipArchiver zipArchiver = new ZipArchiver();
 			zipArchiver.addDirectory(workDir);
-			File cicsBundle = new File(outputDir, project.getArtifactId() + "-" + project.getVersion() + (classifier != null ? "-" + classifier : "") + "." + CICS_BUNDLE_EXTENSION);
+			
+			File cicsBundle = new File(outputDir, finalName + (classifier != null ? "-" + classifier : "") + "." + CICS_BUNDLE_EXTENSION);
```

Comment:
I can't quite see how finalName defaults to
```
project.getArtifactId() + "-" + project.getVersion()
```

What have I missed?

Reply:
Perhaps I've misunderstood - I was expecting you could use that param to override the name, or just leave it out if you want the old behaviour. But it looks like it's a required param now.

Reply:
It's built-in.  Search for `finalName` in here: https://maven.apache.org/pom.html

Reply:
Addresses this issue raised here: https://github.com/IBM/cics-bundle-maven/issues/224

Reply:
It defaults to include the version, but some people may override it to try and remove adding the version to the final artifact's name.  This is just adding support for that.